apps/etterlatte-saksbehandling-ui/client/src/components/behandling/revurderingsoversikt/Revurderingsoversikt.tsx

```diff
@@ -111,7 +111,7 @@ export const Revurderingsoversikt = (props: { behandling: IDetaljertBehandling }
         )}
         <div>
           <strong>Revurdering opprettet: </strong>
-          {format(new Date(behandling.soeknadMottattDato), 'dd.MM.yyyy')}
+          {format(new Date(behandling.soeknadMottattDato!!), 'dd.MM.yyyy')}
```

Comment:
Burde kanskje heller kjørt hvis type revurdering så er denne garantert? (hvis den er det)

Reply:
Grunnen til denne endringa er at vi for migrerte saker ikkje har med søknadsdato.

Veit ikkje korleis det er i andre situasjonar, så har prøvd å ikkje endre på det.

Reply:
Skulle vi bare fjernet revurdering opprettet - evnt endret datoen til noe mer fornuftig? Gir jo ikke mening å vise søknadMottattDato her i utgangspunktet, og hvis vi revurderer en migrert sak her, så vil det vel feile?